src/services/legalEntityManagement/onboardingPage.ts

```diff
@@ -0,0 +1,40 @@
+import getJsonResponse from "../../helpers/getJsonResponse";
+import Service from "../../service";
+import LegalEntityManagementResource from "../resource/legalEntityManagementResource";
+import {OnboardingLinkInfo} from "../../typings/legalEntityManagement/onboardingLinkInfo";
+import {OnboardingLink} from "../../typings/legalEntityManagement/onboardingLink";
+import {ObjectSerializer, OnboardingTheme, OnboardingThemes} from "../../typings/legalEntityManagement/models";
+
+class OnboardingPage extends Service {
+    public async create(id: string, request: OnboardingLinkInfo): Promise<OnboardingLink> {
+        const resource = new LegalEntityManagementResource(this, `/legalEntities/${id}/onboardingLinks`);
+        const response = await getJsonResponse<OnboardingLinkInfo, OnboardingLink>(
+            resource,
+            request,
+            { method: "POST" }
+        );
+        return ObjectSerializer.deserialize(response, "OnboardingLink");
+    }
+
+    public async retrieve(): Promise<OnboardingThemes> {
+        const resource = new LegalEntityManagementResource(this, "/themes");
+        const response = await getJsonResponse<string, OnboardingThemes>(
+            resource,
+            "",
+            { method: "GET" }
+        );
+        return ObjectSerializer.deserialize(response, "OnboardingThemes");
+    }
+
+    public async update(id: string): Promise<OnboardingTheme> {
```

Comment:
This method will `GET` one theme, can you rename it to `retrieve`? Or maybe `retrieveTheme`?